### PR TITLE
Select single recipe

### LINKED ISSUE #5: FE User Story: Recipe Page - View & Navigation
As a user when I click on a specific recipe,
I want to:
- [x] navigate to the `Recipe` page

**view recipe details**
- [x] recipe name
- [ ] image
- [x] approximate cook time
- [x] ingredients
- [x] instructions
- [x] notes

so that I can learn how to make this recipe.

### PR DESCRIPTION
**Pull request by jmevans0211**
## What does this PR do?
- Renders recipe information on the recipe page

### Where should the reviewer start?
- Home page or category page.

### How should this be manually tested?
- Click on the recipe card. You should be taken to the recipe page with relevant information displayed.

### Card(s) closed:
Closes #5 

### Screenshots:
Screen Shot 2020-03-04 at 4 55 45 PM

### Additional notes/questions:
- Don't worry about the `RecipeForm` component for now. It needs some work, that will be in a future PR.
- There is no styling for now, that also will be in a future PR.

### Helpful Resources
-

### CODE REVIEW

File: src/Recipe/Recipe.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  render() {
+    console.log(this.props)
+    let { recipe_name, id, approx_time, instructions, ingredients, notes, image_url, category, category_id } = this.props.recipe
+    return (
+      <div>
+        <Header />
+          <section className='recipe__div--setion'>
+            <div>
+              <h4>{recipe_name}</h4>
+              <p>{category_id}</p>
```
Comment (aripp2) [inline src/Recipe/Recipe.js]: I had to check the code to see what this number was on the page. I am not sure you need to show this to the user.

**Comment by aripp2** [comment]
- Great job getting the individual recipes to show. I would suggest adding a `back` button or option for the user to return to the previous page/view. Other than that, with some styling and formatting you are on the right track!

- I like how you added the ability to navigate back home to view all when clicking on EvansEats.

- I am curious what the `!` is going to be for in the `<footer>`???

- I am looking forward to seeing it as you work on the styling and being able to add a recipe of my own!

- AND the loading gif is fantastic! I know you plan on centering it too!

File: src/Recipe/Recipe.js (hunk 1 above)
Comment (jmevans0211) [reply src/Recipe/Recipe.js]: Ah yes thank you! That is meant to be the `category_name` like "Beverages", "Desserts", etc...

**Comment by jmevans0211** [comment]
@aripp2 Thanks, Amy!

The `!` in the `Footer` component will eventually allow for users to delete a recipe (password protected of course 👍). I'm thinking about adding in a different little symbol/emoji. Maybe a security guard... 💂)

**Pull request merged**